src/main/java/uk/ac/cam/cl/dtg/segue/dao/content/GitContentManager.java

```diff
@@ -438,6 +438,9 @@ public final ResultsWrapper<ContentDTO> questionSearch(
         IsaacSearchInstructionBuilder searchInstructionBuilder = new IsaacSearchInstructionBuilder(
                 searchProvider, this.showOnlyPublishedContent, this.hideRegressionTestContent, !showNoFilterContent)
```

Comment:
The other similar options e.g. `hideRegressionTestContent` are passed in as parameters to the constructor, I'd lean towards doing the same for consistency (or change the lot if there's a good reason to).

Reply:
Yes, I didn't like the endless boolean constructor params, especially given the builder pattern the rest of the class uses. So I made this one consistent with the builder pattern and it would be nice to remove the rest from the constructor but that didn't seem important enough now.